apps/dolly-frontend/src/main/js/src/components/fagsystem/tpsf/form/personinformasjon/partials/sikkerhetstiltak/Sikkerhetstiltak.tsx

```diff
@@ -27,28 +32,57 @@ type Option = {
 export const Sikkerhetstiltak = ({ formikBag }: SikkerhetstiltakProps) => {
 	const opts = useContext(BestillingsveilederContext)
 
+	const navEnheter = SelectOptionsOppslag.hentNavEnheter()
+	const navEnheterOptions = SelectOptionsOppslag.formatOptions('navEnheter', navEnheter)
+
 	const paths = {
-		typeSikkerhetTiltak: 'tpsf.typeSikkerhetTiltak',
-		beskrSikkerhetTiltak: 'tpsf.beskrSikkerhetTiltak',
-		sikkerhetTiltakDatoFom: 'tpsf.sikkerhetTiltakDatoFom',
-		sikkerhetTiltakDatoTom: 'tpsf.sikkerhetTiltakDatoTom',
+		rootPath: 'pdldata.person.sikkerhetstiltak[0]',
+		tiltakstype: 'pdldata.person.sikkerhetstiltak[0].tiltakstype',
+		kontaktperson: 'pdldata.person.sikkerhetstiltak[0].kontaktperson',
+		gyldigFraOgMed: 'pdldata.person.sikkerhetstiltak[0].gyldigFraOgMed',
+		gyldigTilOgMed: 'pdldata.person.sikkerhetstiltak[0].gyldigTilOgMed',
+		tpsMessagingRootPath: 'tpsMessaging.sikkerhetstiltak',
 	}
 
 	const indexBeskrSikkerhetTiltak = 7
 
 	const handleSikkerhetstiltakChange = (option: Option) => {
-		formikBag.setFieldValue(paths.typeSikkerhetTiltak, option.value)
-		formikBag.setFieldValue(
-			paths.beskrSikkerhetTiltak,
-			option.label === 'Opphørt' ? option.label : option.label.substring(indexBeskrSikkerhetTiltak)
+		handleValueChange(option.value, 'tiltakstype')
+		handleValueChange(
+			option.label === 'Opphørt' ? option.label : option.label.substring(indexBeskrSikkerhetTiltak),
+			'beskrivelse'
 		)
 	}
 
+	const handleValueChange = (value: string, path: string) => {
+		formikBag.setFieldValue(`${paths.rootPath}.${path}`, value)
+		formikBag.setFieldValue(`${paths.tpsMessagingRootPath}.${path}`, value)
+	}
+
+	const gyldigFomErDagensDato = () => {
+		return isToday(_get(formikBag.values, 'pdldata.person.sikkerhetstiltak[0].gyldigFraOgMed'))
+	}
+
+	const [randomNavUsers, setRandomNavUsers] = useState([])
+	const [personident, setPersonident] = useState(
+		_get(formikBag.values, 'pdldata.person.sikkerhetstiltak[0].kontaktperson.personident')
+	)
+
+	useEffect(() => {
+		setRandomNavUsers(genererTilfeldigeNavPersonidenter())
+	}, [])
+
 	return (
 		<Vis attributt={Object.values(paths)} formik>
 			<div className="flexbox--flex-wrap">
+				{!gyldigFomErDagensDato() && (
+					<AlertStripeAdvarsel>
```

Comment:
Vet ikke om denne like greit kunne vært et varsel under dato-feltet? Føler den havner litt langt unna "der det skjer" 🤔  Men dette er bare synsing da 😁  Om du ser på inntektsmelding og velger foreldrepenger på ytelse får man en advarsel på startdato for periode (som ikke hindrer deg i å gå videre). Kunne noe sånt funket?